ckan/tests/lib/test_helpers.py

```diff
@@ -635,7 +635,7 @@ def test_active_in_resource_controller(self, test_request_context):
         with test_request_context(u'/dataset/' + resource['package_id'] + '/resource/' + resource['id']):
             menu = (
                 ("home.index", "Home"),
-                ("dataset.search", "Datasets", ['dataset', 'resource']),
+                ("dataset.search", "Datasets", ['dataset', 'resource', 'dataset_resource']),
```

Comment:
what's this change for?

Reply:
@wardi because it has `with test_request_context(u'/dataset/' + resource['package_id'] + '/resource/' + resource['id'])`, with the latest changes, it is going to load the `dataset_resource` blueprint as a priority. So the `dataset_resource` addition is to just highlight that blueprint if it is there, giving it the active class to match in the test.